src/KeyVault/KeyVault/Commands/NewAzureKeyVault.cs

```diff
@@ -28,6 +28,9 @@ namespace Microsoft.Azure.Commands.KeyVault
     /// <summary>
     /// Create a new key vault.
     /// </summary>
+    [GenericBreakingChange("Warning! The ability to create new key vaults with soft delete disabled will be deprecated by December 2020. " +
```

Comment:
```suggestion
    [GenericBreakingChange("The ability to create new key vaults with soft delete disabled will be deprecated by December 2020. " +
```
I suppose we can remove the leading "Warning!" because PowerShell already prints a "Warning:" for us